src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -237,22 +240,63 @@
             <argument type="service" id="sylius.repository.payment_method" />
         </service>
 
-        <service id="sylius.taxation.order_shipment_taxes_by_zone_applicator" class="%sylius.taxation.order_shipment_taxes_by_zone_applicator.class%">
+        <service id="sylius.taxation.order_shipment_taxes_applicator" class="%sylius.taxation.order_shipment_taxes_applicator.class%">
```

Comment:
E.g. you can use some external service for tax calculation that can identify the rates using country/province (that can be taken from order entity) and know nothing about the Sylius' zones. I would at least leave `byZone` in the interfaces (e.g. [here](https://github.com/Sylius/Sylius/pull/4406/files#diff-cb3e1d18d47a2ce77e48662ae8172afdL20)), wdyt?